pkg/plugin/v3/scaffolds/internal/templates/main.go

```diff
@@ -146,6 +146,11 @@ func (f *MainUpdater) GetCodeFragments() file.CodeFragmentsMap {
 		return fragments
 	}
 
+	// If Group is empty add the core in import path
+	if f.Resource.Group == "" {
+		f.Resource.Group = "core"
+	}
+
 	// Generate import code fragments
 	imports := make([]string, 0)
 	imports = append(imports, fmt.Sprintf(apiImportCodeFragment, f.Resource.ImportAlias, f.Resource.Package))
```

Comment:
We need to add this scenario at https://github.com/kubernetes-sigs/kubebuilder/blob/master/generate_testdata.sh as well in order to generate for the v3 testdata projects an API with a group empty too.

Reply:
Done

Reply:
You can close this one it is solved.